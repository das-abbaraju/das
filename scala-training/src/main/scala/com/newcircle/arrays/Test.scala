package com.newcircle.arrays

/**
 * Created by dasabbaraju on 01/12/14.
 */
class Test extends  App {

  def increment(start: Int, finish: Int) = {
    for (i <- start to finish) {
      println("Current value (increasing from " + start + " to " + finish + ") is " + i)
    }
  }

  object Test {
    def main(args:Array[String]) = {
      println("Program execution start.")

      println("Program execution end.")
    }
  }



}
