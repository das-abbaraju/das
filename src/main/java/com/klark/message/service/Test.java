// =======================================================
// Copyright Mylife.com Inc., 2014. All rights reserved.
//
// =======================================================

package com.klark.message.service;

import com.klark.user.model.UserMessage;

/**
 * Description here!
 * 
 * 
 * @author
 */

public class Test<T extends UserMessage> {

    private T t;

    public void set(T t) {
        this.t = t;
    }

    public T get() {
        return t;
    }

    public String getString() {
        return t.getClass().toString();
    }

    public <U extends Number> void inspect(U u) {
        System.out.println("T: " + t.getClass().getName());
        System.out.println("U: " + u.getClass().getName());
    }

    public static void main(String[] args) {
        Test<UserMessage> integerBox = new Test<UserMessage>();
        // integerBox.set(new Integer(10));
        // integerBox.inspect(null); // error: this is still String!
    }
}