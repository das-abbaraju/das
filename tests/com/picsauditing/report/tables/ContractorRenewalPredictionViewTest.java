package com.picsauditing.report.tables;

import org.approvaltests.Approvals;
import org.approvaltests.reporters.DiffReporter;
import org.approvaltests.reporters.UseReporter;
import org.junit.Test;

@UseReporter(DiffReporter.class)
public class ContractorRenewalPredictionViewTest {

    @Test
    public void testSQL() throws Exception {
        ContractorRenewalPredictionView view = new ContractorRenewalPredictionView();
        Approvals.verify(view.toString());
    }
}
