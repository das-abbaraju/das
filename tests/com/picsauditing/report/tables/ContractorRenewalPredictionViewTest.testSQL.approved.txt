(SELECT accounts.id AS conID, ROUND(100 * (0.16065856 + (0.08340467 * AVG(CASE generalcontractors.flag WHEN 'Green' THEN 3 WHEN 'Amber' THEN 2 ELSE 1 END))
                       + (0.20098547 * CASE contractor_info.paymentMethod WHEN 'Check' THEN 1 ELSE 0 END)
                       + (0.461150415 * IFNULL(contractor_info.ccOnFile,0))
                       + (0.065215233 * IFNULL(contractor_info.ccExpiration > contractor_info.paymentExpires,0))
                       + (0.050759683 * CASE WHEN payingFacilities <= 1 THEN 1 WHEN payingFacilities <= 2 THEN 2 WHEN payingFacilities <= 12 THEN 3 ELSE 3.5 END)
                      )) AS predictor
                      FROM accounts
                      JOIN contractor_info ON contractor_info.id = accounts.id
                      JOIN generalcontractors ON generalcontractors.subID = contractor_info.id
                      JOIN accounts clients ON clients.id = generalcontractors.genID AND clients.type = 'Operator'
                      GROUP BY accounts.id )